src/java.base/share/classes/jdk/internal/access/foreign/MemorySegmentProxy.java

```diff
@@ -32,26 +32,26 @@
  * This proxy interface is required to allow instances of the {@code MemorySegment} interface (which is defined inside
```

Comment:
"This abstract base class.."? 

I don't mind the current name, but should the class be renamed `AbstractMemorySegmentProxy`?

Reply:
I'll fix the doc - since this is a transitional artifact (will disappear when API is finalized an in java.base) I'd prefer to avoid the renaming